serverless/configuring-knative-serving-autoscaling.adoc

```diff
@@ -0,0 +1,13 @@
+[id="configuring-knative-serving-autoscaling"]
+= Configuring Knative Serving autoscaling
+include::modules/common-attributes.adoc[]
+:context: configuring-knative-serving-autoscaling
+
+Red Hat OpenShift Serverless provides capabilities for automatic pod scaling, including scaling inactive pods to zero, by enabling the Knative Serving autoscaling system in an {product-title} cluster.
+
+To enable autoscaling for Knative Serving, you must configure concurrency and scale bounds in the revision template.
+
+NOTE: Any limits or targets set in the revision template are measured against a single instance of your application. For example, setting the `target` annotation to `50` will configure the autoscaler to scale the application so that each instance of it will handle 50 requests at a time.
```

Comment:
maybe highlight "each instance of it"

Reply:
Shouldn't really be necessary when it's already in a note and it would look weird vs. OpenShift docs formatting if we randomly make something bold etc. I'd discourage doing this in general for accessibility reasons also, people should just read the note.